addon/-scheduler.js

```diff
@@ -23,7 +23,7 @@ const Scheduler = EmberObject.extend({
     this.queuedTaskInstances = [];
   },
 
-  cancelAll(reason) {
+  cancelAll({ reason }) {
```

Comment:
Any reason (lol) this needed to be changed?

Reply:
Not really. As mentioned in my comment, I did this for internal consistency (and when I started with the signature change, I honestly wasn't sure which implementation of `cancelAll` would get the `resetState` logic).

I would be happy to revert the signature change of `scheduler.cancelAll` if you like. What do you think?

Reply:
Yeah it's not a big thing, but since `cancelAll` on the scheduler is pretty private, I'd prefer a smaller diff.

Reply:
I will make the change :)